models/models.reviews.js

```diff
@@ -34,3 +34,26 @@ exports.selectReviewById = (reviewId) => {
 		return reviews[0];
 	});
 };
+
+exports.updateReviewById = (newVotes, reviewId) => {
+	// Check for 204
+	if (newVotes.inc_votes === 0) {

```

Comment:
newVotes.inc_votes isn't a very semantic way of describing this, I would be tempted to either rename the variable to something clearer or to destructure the inc_votes property